mavisp/mavisp.py

```diff
@@ -222,11 +222,14 @@ def main():
 
 
     out_table = mfs.dataset_table[mfs.dataset_table.apply(lambda r: len(r['criticals']) == 0, axis=1)]
-    out_table = out_table[['system', 'uniprot_ac', 'refseq_id','review_status', 'mode', 'curators']]
+    out_table = out_table[['system', 'uniprot_ac', 'refseq_id','ensemble_source','ensemble_size_foldx','ensemble_size_rosetta','review_status', 'mode', 'curators']]
```

Comment:
can we move `mode` to after `refseq_id`? So the reader first reads the mode and then the ensembles if any